include/alp/amf-based/storagebasedmatrix.hpp

```diff
@@ -167,7 +167,7 @@ namespace alp {
 			/** Get the reference to the AMF of a storage-based matrix */
 			template<
 				typename MatrixType,
-				std::enable_if< internal::is_storage_based< MatrixType >::value > *
+				std::enable_if_t< internal::is_storage_based< MatrixType >::value > *
```

Comment:
missing `= nullptr`

Reply:
Cannot use default in template friend declaration